src/frontend/src/app/pages/login/login-dialog/login-dialog.component.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { LoginDialogComponent } from './login-dialog.component';
+
+describe('HomeDialogComponent', () => {
```

Comment:
apenas um citação: esses arquivos de teste no momento são inúteis, porém não estou sugerindo apagar pq espero que vc os use pra fazer os testes desses components